packages/api/lib/ems.js

```diff
@@ -101,6 +102,22 @@ async function getExpiredS3Objects(bucket, prefix, retentionInDays) {
     .map((s3Object) => ({ Bucket: bucket, Key: s3Object.Key }));
 }
 
+async function retrievePrivateKey() {
+  const privateKeyFile = process.env.ems_privateKey || 'ems-private.pem';
+
+  let privateKey;
+  try {
+    privateKey = await getTextObject(process.env.system_bucket, `${process.env.stackName}/crypto/${privateKeyFile}`);
+  } catch (e) {
+    if (e.code === 'NoSuchKey') {
+      throw new Error(`${privateKeyFile} does not exist in S3 crypto directory`);
```

Comment:
It might be worth including the `s3://bucket/key` here, so it's clear what file didn't exist.